elements/dat-import.js

```diff
@@ -64,19 +64,14 @@ function datImportElement (props) {
 
   assert.equal(typeof onsubmit, 'function', 'dat-import: onsubmit should be type function')
 
-  const linkIcon = icon({
-    id: 'link',
-    cls: 'absolute top-0 bottom-0 left-0'
-  })
-
   return html`
     <label for="dat-import" class="relative dib pa0 b--none ${prefix}">
       <input name="dat-import"
         type="text"
         placeholder="Import dat"
         onkeydown=${onKeyDown}
         class="input-reset">
-      ${linkIcon}
+      ${icon('link', { class: 'absolute top-0 bottom-0 left-0' })}
```

Comment:
Yeah nah, agree - this would only work for icons right now; buttons is harder - gonna leave this for a bit